ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/IdentifiableState.kt

```diff
@@ -0,0 +1,15 @@
+package net.corda.v5.ledger.utxo
+
+import java.util.*
```

Comment:
I think detekt does not like `*`imports.

Reply:
Detekt is passing locally (as is the build). I think detekt is okay with wildcard imports for java libraries, but fails for stuff we add to the API.